__tests__/app.test.js

```diff
@@ -121,9 +122,46 @@ describe("GET Endpoints", () => {
   });
 });
 
+describe("POST Endpoints", () => {
+  
+  describe("POST: /api/articles/:article_id/comments (07)", () => {
+    const newComment = {
+      username: 'rogersop',
+      body: 'Some new comment for article with id: 3',
+    };
+
+    it('Adds a new comment to article, if article exists', () => {
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+          const { comment } = response.body;
+          expect(comment).toBeInstanceOf(Object); //Array of comment objects
+          expect(comment.article_id).toBe(3);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+          });
+        });
+    });
```

Comment:
the comment you post will be the comment you get back. have a think about what you can expect